src/labeler.ts

```diff
@@ -22,69 +23,69 @@ export async function run() {
     const syncLabels = !!core.getInput('sync-labels');
     const dot = core.getBooleanInput('dot');
 
-    const prNumber = getPrNumber();
-    if (!prNumber) {
+    const prNumbers = getPrNumbers();
+    if (!prNumbers) {
       core.info('Could not get pull request number from context, exiting');
       return;
     }
 
     const client: ClientType = github.getOctokit(token, {}, pluginRetry.retry);
 
-    const {data: pullRequest} = await client.rest.pulls.get({
-      owner: github.context.repo.owner,
-      repo: github.context.repo.repo,
-      pull_number: prNumber
-    });
-
-    core.debug(`fetching changed files for pr #${prNumber}`);
-    const changedFiles: string[] = await getChangedFiles(client, prNumber);
-    const labelGlobs: Map<string, StringOrMatchConfig[]> = await getLabelGlobs(
-      client,
-      configPath
-    );
-
-    const prLabels: string[] = pullRequest.labels.map(label => label.name);
-    const allLabels: Set<string> = new Set(prLabels);
-
-    for (const [label, globs] of labelGlobs.entries()) {
-      core.debug(`processing ${label}`);
-      if (checkGlobs(changedFiles, globs, dot)) {
-        allLabels.add(label);
-      } else if (syncLabels) {
-        allLabels.delete(label);
-      }
-    }
-
-    const labels = [...allLabels].slice(0, GITHUB_MAX_LABELS);
-    const excessLabels = [...allLabels].slice(GITHUB_MAX_LABELS);
-
-    try {
-      if (!isListEqual(prLabels, labels)) {
-        await setLabels(client, prNumber, labels);
+    for (const prNumber of prNumbers) {
+      const {data: pullRequest} = await client.rest.pulls.get({
```

Comment:
We should add there a try catch with clarifying message in case of error. Otherwise it throws ambiguous "Not found". [Example](https://github.com/actions/labeler/blob/main/src/labeler.ts#L74)